src/coreComponents/dataRepository/Group.cpp

```diff
@@ -250,8 +250,7 @@ void Group::processInputFile( xmlWrapper::xmlNode const & targetNode,
                      GEOS_FMT( "Error in {}: XML Node at '{}' contains unused attribute '{}'.\n"
                                "Valid attributes are:\n{}\nFor more details, please refer to documentation at:\n"
                                "http://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/userGuide/Index.html",
-                               getDataContext(), targetNode.path(), attributeName,
-                               dumpInputOptions() ),
+                               targetNode.path(), attributeName, dumpInputOptions() ),
```

Comment:
this one will crash ! 

```suggestion
                     GEOS_FMT( "XML Node at '{}' contains unused attribute '{}'.\n"
                               "Valid attributes are:\n{}\nFor more details, please refer to documentation at:\n"
                               "http://geosx-geosx.readthedocs-hosted.com/en/latest/docs/sphinx/userGuide/Index.html",
                               targetNode.path(), attributeName, dumpInputOptions() ),
```